### PR TITLE
Integration with Portale nieruchomości

### PR DESCRIPTION
**Pull request by MonikaKurowska**
This commit adds integration with Portale Nieruchomosci.
This integration requires Newtonsoft.Json NuGet package and adds it to Application module.

### CODE REVIEW

File: Application/Portalenieruchomosci/PortaleIntegration.cs [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                }
+
+                var street_name = offer["street_name"];
+                var building_number = offer["house_number"];
+
+                if (street_name != null)
+                    adr.StreetName = street_name.ToString();
+                if (building_number != null)
+                    adr.DetailedAddress = building_number.ToString();
+                entry.PropertyFeatures = new PropertyFeatures();
```
Comment (PiotrTybura) [inline Application/Portalenieruchomosci/PortaleIntegration.cs:160]: Theoritcally we can read rest of PropertyFeatures from raw description - by parsing it

**Comment by PiotrTybura** [comment]
Right now, this PR is rated for 80% of available points. If you would like to get 100% - Please implement a thing mentioned in the comment + extract this project to external library - it should still work with the most recent commit that I added yesterday morning to the master branch

File: Application/Portalenieruchomosci/PortaleIntegration.cs (hunk 1 above)
Comment (MonikaKurowska) [reply Application/Portalenieruchomosci/PortaleIntegration.cs]: I have added detection of other features from raw description

**Comment by MonikaKurowska** [comment]
> extract this project to external library - it should still work with the most recent commit that I added yesterday morning to the master branch

Done

**Comment by PiotrTybura** [comment]
Project graded for 100%.

**Pull request closed without merging**